src/control/pure_pursuit_cpp/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>pure_pursuit_cpp</name>
+    <version>0.0.1</version>
+    <description>Pure Pursuit in C++</description>
+    <maintainer email="team@qutmotorsport.com">Alistair English, Zac Gillerat</maintainer>
```

Comment:
English isn't here anymore either ;)

Reply:
Whoops, copied yaw controller and thought that was you.